demo/admin/src/products/future/generated/ManufacturersGrid.tsx

```diff
@@ -137,7 +138,27 @@ export function ManufacturersGrid(): React.ReactElement {
         },
         {
             field: "address_alternativeAddress_street",
-            headerName: intl.formatMessage({ id: "manufacturer.address.alternativeAddress.street", defaultMessage: "Alt-Street" }),
+            renderHeader: () => (
+                <>
+                    <GridColumnHeaderTitle
+                        label={intl.formatMessage({ id: "manufacturer.address.alternativeAddress.street", defaultMessage: "Alt-Street" })}
+                        columnWidth={150}
+                    >
+                        {intl.formatMessage({ id: "manufacturer.address.alternativeAddress.street", defaultMessage: "Alt-Street" })}
+                    </GridColumnHeaderTitle>
+                    <Tooltip
+                        trigger="hover"
+                        title={
+                            <FormattedMessage
+                                id="manufacturer.address.alternativeAddress.street.tooltip"
+                                defaultMessage="Street of alternative address"
+                            />
+                        }
+                    >
+                        <Info sx={{ margin: 1 }} />
```

Comment:
Shouldn't this be margin-left only? @jamesricky is this the preferred way?

Reply:
Yes, since it's vertically centered, that should be enough

```suggestion
                        <Info sx={{ marginLeft: 1 }} />
```